app/src/app/[lng]/data/[step]/SourceDrawer.tsx

```diff
@@ -95,16 +95,15 @@ export function SourceDrawer({
               </Stack>
             </DrawerBody>
           )}
+          <Stack
+            w="full"
+            className="drop-shadow-top border-t-2 absolute left-0 flex justify-center items-center"
```

Comment:
Keeping the padding might be good in case the width of the source drawer is lower than the hardcoded button width.